.travis.yml

```diff
@@ -5,7 +5,9 @@ rvm:
   - 2.3.3
   - jruby-9.1.6.0
   - jruby-head
-before_install: gem install bundler
+before_install:
+  - gem update --system
```

Comment:
This was needed in order to fix a `can't modify frozen String` error on the travis builds when installing the `rainbow` gem. It's unrelated to the rest of my PR. (See bundler/bundler#5357)